### PR TITLE
Create 01-why-react.de-DE.md

### PR DESCRIPTION
**Pull request by thomasp9**
Translation of 01-why-react.de-DE.md to German

### CODE REVIEW

File: docs/docs/01-why-react.de-DE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: Warum React?
+permalink: why-react-de-DE.html
+---
+React ist eine JavaScript-Bibliothek von Facebook und Instagram für Benutzeroberflächen. Man kann sich React als das **V** in **[MVC](https://de.wikipedia.org/wiki/Model_View_Controller)** vorstellen.
+
+Wir haben React für folgende Aufgabe entwickelt: **umfangreiche Anwendungen zu bauen, deren Daten sich zeitlich ändern**.
+
+## Einfach
+
+Beschreibe, wie sich Deine App zu jedem gewünschten Zeitpunkt präsentieren soll und React kümmert sich um alle Benutzeroberflächen-Änderungen sobald sich die zugrundliegenden Daten ändern.
```
Comment (jimfb) [inline docs/docs/01-why-react.de-DE.md]: @thomasp9 Would like to start closing out PRs like this.  Did you want to update this PR to fix the zugrundeliegenden typo and then we can merge, or should we close this PR, or is it fine as-is,, or how do you want to handle this?

Comment (thomasp9) [inline docs/docs/01-why-react.de-DE.md]: Hi

I wanted to fix the typo myself but honestly, this is my first time ever doing anything on git and I am unsure how to go about this edit.

After that fix, text is ready to be merged. Thanks for reaching out.

Regards
Patrick Thomas 

Von meinem iPhone gesendet

Comment (jimfb) [inline docs/docs/01-why-react.de-DE.md]: If you already have the branch checked out, you can just make the change and use `git commit --amend` to update the old commit.  (If you don't have it checked out, look at https://help.github.com/articles/checking-out-pull-requests-locally/)

Then you can use `git push -f` to forcefully update the branch.

**Comment by thomasp9** [comment]
Finally came around to fix this aka found the right tab to edit the file online.

**Comment by zpao** [comment]
Squashed and merged. @thomasp9 Thanks!

**Comment by zpao** [comment]
d17ad0f0c0ef44371a260964b9b8f6e45d3eeace

**Pull request closed without merging**